src/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/InteropHelpers.cs

```diff
@@ -316,6 +316,42 @@ internal static unsafe void FixupMethodCell(IntPtr hModule, MethodFixupCell* pCe
             }
         }
 
+#if !PLATFORM_UNIX
```

Comment:
I'm not sure if there's a Windows define, but it might be good to use that if there is since we might want non-Windows non-Unix platforms (perhaps WASM) at some point.